src/App/Entity/Meetup.php

```diff
@@ -63,7 +63,12 @@
     private $talks;
 
     /**
-     * @ORM\OneToMany(targetEntity=MeetupAttendee::class, mappedBy="meetup")
+     * @ORM\OneToMany(
+     *     targetEntity=MeetupAttendee::class,
+     *     mappedBy="meetup",
+     *     orphanRemoval=true,
+     *     cascade={"persist", "remove"}
```

Comment:
orphan removal is actually what fixes this issue. Probably don't need the cascade